types.ts

```diff
@@ -86,6 +93,7 @@ export type ContextData = {
   reqHeaders?: {
     authorization?: string;
     origin?: string;
+    sessionid?: string;
```

Comment:
just for minor styling - https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive#:~:text=Header%20names%20are%20not%20case,%22)%20and%20the%20field%20value.

```suggestion
    sessionId?: string;
```